### ISSUE TITLE
Add ember-data imports snippet

### ISSUE THREAD

**Issue by Exelord** [mattmcmanus/atom-ember-snippets#46]
We should add snippets for all those modern imports:
- `immodel`
- `imadapter`
- `imtransform`
- `imserializer`
- `imattr`
- `imhasMany`
- `imbelongsTo`

```
import Model from 'ember-data/model';
import Adapter from 'ember-data/adapter';
import Transform from 'ember-data/transform';
import Serializer from 'ember-data/serializer';

import attr from 'ember-data/attr';
import { hasMany } from 'ember-data/relationships';
import { belongsTo } from 'ember-data/relationships';
```

**Comment by mattmcmanus** [comment]
Hey @Exelord, Thanks for creating all the issues. I was just thinking last night that things need to be updated. I'll try to get around to it soon. I'd happily accept PRs as well!

**Comment by Exelord** [comment]
Cool! :) Well, I dont have too much time right now, but I just wanted to point this out in case someone would like to help.

**Comment by mattmcmanus** [comment]
Great. Thanks!

On Wed, Aug 29, 2018, at 12:43 PM, Maciej Kwaśniak wrote:
> Cool! :) Well, I dont have too much time right now, but I just wanted
> to point this out in case someone would like to help.> — You are receiving this because you commented. Reply to this email
> directly, view it on GitHub[1], or mute the thread[2].

Links:

  1. https://github.com/mattmcmanus/atom-ember-snippets/issues/46#issuecomment-417021845

**Comment by Exelord** [comment]
Closed in favor of #47

**Comment by mattmcmanus** [comment]
Published in `3.0.0`!
